pkg/allocateip/allocateip.go

```diff
@@ -8,79 +8,228 @@ import (
 	"reflect"
 	"time"
 
+	log "github.com/sirupsen/logrus"
```

Comment:
I think we've been trying to avoid aliasing `logrus` to `log` elsewhere. Do we need to do this?

Reply:
I guess it's fine. I see we were importing it as both before. But I would have removed the aliased one rather than the other way around /shrug

Reply:
I've no issue switching it up - didn't realize we were avoiding log.

Reply:
We're just not consistent with it. Sometimes we alias, sometimes we declare a logger instance named `log`. It's just all over the place, so I wouldn't worry about it..